plugins/backbones/threatbus_inmem/plugin.py

```diff
@@ -64,5 +58,5 @@ def run(config, logging, inq):
         validate_config(config)
     except Exception as e:
         logger.fatal("Invalid config for plugin {}: {}".format(plugin_name, str(e)))
-    threading.Thread(target=provision, args=(logger, inq), daemon=True).start()
+    threading.Thread(target=provision, args=(inq,), daemon=True).start()
```

Comment:
Is that valid syntax?

Reply:
Yes, valid and even required. The `args` must be passed as tuple but it would not be interpreted at tuple without the `,` .